docs/components/aws/elasticache.md

```diff
@@ -0,0 +1,792 @@
+required.
+
+#### Cluster Engine Configuration
+
+<p align="center">
+  <img style="Padding:20px" width="400" src="./images/elasticache/ecache_cluster_conf.png">
+</p>
+
+Field                       | Required           | Description
+---                         | :---:              | ---
+Engine                      | :white_check_mark: | The cache engine type to be used for the cluster   
+Engine Version              | :white_check_mark: | The version number of the cache engine to be used for the cluster 
+Node Type                   | :white_check_mark: | The compute and memory capacity of the node
+Cluster mode                | :white_check_mark: | Standalone or Clustered 
+Multi-AZ with Auto-failover | :white_check_mark: | Auto failover with multi availability zones
+Number of Shards            | :white_check_mark: | Number of shards in the cluster 
+Number of Replicas          | :white_check_mark: | Number of replicas within a single shard
+&nbsp;
+
+Field                       | Constraints
+---                         | ---
+Multi-AZ with Auto-failover | Only available in standalone mode(cluster mode disabled). In cluster mode enabled by default
+Number of Shards            | Only available in cluster mode, minimum:2 maximum:15. Otherwise, default to be 1  
+Number of Replicas          | Maximum: 6, If Multi-AZ with Auto-failover enabled minimum should be 2, 1 otherwise
+&nbsp;
+
+#### Cluster Settings
+
+<p align="center">
+  <img style="Padding:20px" width="400" src="./images/elasticache/ecache_cluster_settings.png">
+</p>
+  
+
+Field               | Required           | Description
+---                 | :---:              | ---
+Cluster Name        | :white_check_mark: | Name for the cluster to be used as an identifier 
+Cluster Description | :white_check_mark: | Description for the cluster
+Port                | :white_check_mark: | The port number on which cluster accepts connections
+&nbsp;
+
+Field        | Constraints
+---          | ---
+Cluster Name | Name must contain from 1 to 20 alphanumeric characters or hyphens, the first character must be a letter, the name cannot end with a hyphen or contain two consecutive hyphens.
+&nbsp;
+
+#### VPC Configuration
+
+When creating a new ElastiCache cluster, you need to configure a VPC as well. Sigma enables
+functionality to create new VPC as well as use an existing one.
+
+##### Create New VPC
+
+To create new VPC, expand VPC settings and select **Create New VPC** option from VPC type dropdown.
+
+<p align="center">
+  <img style="Padding:20px" width="400" src="./images/elasticache/ecache_new_vpc.png">
+</p>
+
+Field                 | Required           | Description
+---                   | :---:              | ---
+VPC Name              | :white_check_mark: | Name of the VPC 
+CIDR block            | :white_check_mark: | IPv4 CIDR block for the VPC
```

Comment:
better to link if there are any AWS docs (same for subnet and its CIDR block, below)